components/FeatureHighlight/examples/FeatureHighlightTypicalUseViewController.m

```diff
@@ -44,7 +44,7 @@ - (void)didTapButton:(id)sender {
   vc.bodyText = @"This is the description of the feature highlight view controller.";
   // TODO(https://github.com/material-components/material-components-ios/issues/3644 ):
   // Disable the incorrect "Double tap" hint for now
-  vc.accessibilityHint = nil;
+  vc.accessibilityLabel = @"Try out this new feature";
```

Comment:
I think you mixed-up the `accessibilityHint` and `accessibilityLabel`. We should continue to nil-out the hint so that the "double tap to dismiss" isn't provided to users.